browser-test/src/applicant/questions/radio_button.test.ts

```diff
@@ -34,6 +36,11 @@ test.describe('Radio button question for applicant flow', () => {
       await logout(page)
     })
 
+    test.beforeEach(async () => {
+      const {page} = ctx
```

Comment:
recommendation: Highly recommend not setting the flag in the beforeEach when it gets set to a different value later. It can make it much more challenging to know for certain which  state it is in within a test.

If you want to stick with a beforeeach for now you could put the new tests requiring it be enabled into a higher level  describe, a sibling to 'Radio button question for applicant flow'. Because of our current way of using the page context you'll still need to make sure the flag is enabled/disable as needed, but it'll keep things more separated which is something The Offspring would be proud of.

Reply:
split into a separate describe